### PR TITLE
cocartesian fibrations

### LINKED ISSUE #25: (meta-)bicategories
I have been thinking a bit about how to deal with strictness and weakness in these notes. I want to avoid getting too arcane and religious about things, but at the same time I need just enough to set up the theory properly.

Consequently I think that it makes sense to extend and adjust the `Foundational Assumptions` section to define the concept of a "(meta-)bicategory", and then say that `CAT` is not the metacategory of all categories but rather the meta-bicategory of all categories. This is honestly needed already in order to provide intuition for what a cartesian fibration is.

Then later on under internal categories, we can define for any category `C` the *category* `Cat(C)` of categories internal to `C`. In particular when `C = Set`, this gives us an appropriate basis for the theory of strict categories, split fibrations, strictly discrete fibrations, etc.

We can get get a good rosetta stone now:

1. Cartesian fibration = pseudofunctor `C^op -> CAT`
2. Split cartesian fibration = functor `C^op -> Cat(SET)`
3. Right fibration = pseudofunctor `C^op -> GPD`
4. Discrete right fibration = pseudofunctor `C^op -> 0-Trunc(GPD)`
5. Strictly discrete right fibration = functor `C^op -> SET`.

@cangiuli What do you think about this?

### PR DESCRIPTION
**Pull request by cangiuli**
Ready to merge once we:
- [x] rename the new nodes
- [x] come up with a better notation for the opposite fibration

This section is sorely in need of a remark about pseudofunctors `C -> CAT`, but I don't think it's blocking. Once we fix #25, we should pepper such remarks throughout the entire document.

### CODE REVIEW

File: _nodes/0001.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ macrolib: topos
 
 Let $E$ be displayed over $B$, and let $f:x\to y \in B$; a morphism
 $\bar{f}:\bar{x}\to\Sub{f} \bar{y}$ in $E$ is called *cartesian* over $f$ when for
-any $m:u\to x$ and $\bar{h}:\bar{u}\to\Sub{u;f} \bar{y}$ there exists a unique
+any $m:u\to x$ and $\bar{h}:\bar{u}\to\Sub{m;f} \bar{y}$ there exists a unique
```
Comment (cangiuli) [inline _nodes/0001.md:8]: somehow this typo has been here all along :laughing:

File: _nodes/0021.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+title: Opcartesian morphisms
```
Comment (jonsterling) [inline _nodes/0021.md]: can we call it cocartesian instead?

Comment (cangiuli) [reply _nodes/0021.md]: And cocartesian fibration? Sure.

Comment (cangiuli) [reply _nodes/0021.md]: Well it's a tiny bit awkward because people shy away from "cofibration" for obvious reasons, but I don't feel strongly.

Comment (jonsterling) [reply _nodes/0021.md]: right. i don’t want to say cofibration. i think it is just very common in this school to say cocartesian.

Comment (cangiuli) [reply _nodes/0021.md]: Indeed, I just mean that "cocartesian fibration" is more different to "opfibration" than "opcartesian fibration" is, but I don't mind for any reason other than potential confusion.

File: _nodes/0001.md (hunk 1 above)
Comment (jonsterling) [reply _nodes/0001.md:8]: 🤦🏼

File: _nodes/0023.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+title: Opposite fibrations
```
Comment (jonsterling) [inline _nodes/0023.md]: I think we need to call this something different (and also rename the macro, or perhaps not even have a macro).

Comment (cangiuli) [reply _nodes/0023.md]: Yes, please suggest something!

File: _nodes/0024.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+---
+title: "Example: Coslices and pushouts"
```
Comment (jonsterling) [inline _nodes/0024.md]: Huh, cool. Somehow I didn't know about this LOL

File: _nodes/0023.md (hunk 3 above)
Comment (jonsterling) [reply _nodes/0023.md]: "Global opposite" or "Total opposite"?

File: _nodes/0023.md [hunk 5]
```diff
@@ -1,23 +1,28 @@
...
+1. An object of $\TotOpCat{E}\Sub{x}$ is given by an object of $E\Sub{x}$.
 
-2. Given $f : x \to y\in \OpCat{B}$, a displayed morphism $\bar{x}\to\Sub{f} \bar{y}$ in $\OpFib{E}$
+2. Given $f : x \to y\in \OpCat{B}$, a displayed morphism $\bar{x}\to\Sub{f} \bar{y}$ in $\TotOpCat{E}$
    is given by a displayed morphism $\bar{y}\to\Sub{f} \bar{x}$ in $E$.
 
 **Warning.** Do not confuse this construction with {%ref 000Q %}, which produces a
 displayed category over $B$.
 
+**Exercise.** Let $E$ be displayed over $B$. Prove that the total category {%ref 000A %}
```
Comment (cangiuli) [inline _nodes/0023.md]: "actually, this strange notation is very good"

**Review by jonsterling — COMMENTED** [review]
total landscaping

**Pull request merged**